src/test/java/io/jenkins/pluginhealth/scoring/service/PluginServiceIT.java

```diff
@@ -1,3 +1,27 @@
+/*
```

Comment:
This file was originally created by @dheerajodha. Do you agree to this license here?

Reply:
I agree.